code-challenges.test.js

```diff
@@ -13,22 +13,63 @@
 // --------------------1) Prompt: Create a function that takes in a number (greater than 2) and returns an array containing the Fibonacci sequence. The length of the array is determined by the argument of the function.
 
 // a) Create a test with expect statements for each of the variables provided.
+describe('Fibonacci Sequence Generator', () => {
+  it('should generate the Fibonacci sequence with a length of 6', () => {
+  const fibonacciLength1 = 6
+  // Expected output: [1, 1, 2, 3, 5, 8]
+  const fibonacciLength2 = 10
+  // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+ expect(generateFibonacci(fibonacciLength1)).toEqual([1, 1, 2, 3, 5, 8])
+ expect(generateFibonacci(fibonacciLength2)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
+ })
+}) 
 
-const fibonacciLength1 = 6
-// Expected output: [1, 1, 2, 3, 5, 8]
+//ReferenceError: FibonacciSequenceGenerator is not defined- good failure
 
-const fibonacciLength2 = 10
-// Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
 // b) Create the function that makes the test pass.
 
 // Pseudo code:
+//create function named generateFibonacci 
+//use if to check length is less than two
+// return "Length should be greater than or equal to 2"
+//create array called fibonacci and add 1 to the beginning
+//use for to check i and the lengths
+//add the sum of the last two numbers in the array
+//add nextFibonacci to the end of the fibonacci array
+//return fibonacci array
```

Comment:
Line 37 doesn't make sense, what do you mean check i and the lengths?
Here is how you can rewrite this:
- create variable holding array with 1 and 1 as values
- iterate using for loop starting at index 2
- each iteration adds the values at the previous 2 indices then adds the sum to the created array

Reply:
yeah i definitely been trying to work on my pseudo code and explaining that better.